index.html

```diff
@@ -229,22 +229,20 @@ <h5>Skills</h5>
                 <hr/>
 			</div>
     </section>
-	
-	<!-- the contact form's section -->
-<section class="form">
-  <div class="contact-sec">
-    <h2>Contact me</h2>
-     <p>If you have an application you <br> are interested in developing,a <br> feature that you need built or a <br> project that needs coding.I'd <br> love to help with it.</p> 
-  </div>
-<form action="https://formspree.io/f/mzbowdel" method="post">
-  <ul>
-    <li><input type="text" class="name" id="name" maxlength="30" placeholder="Tshepo Yusuf" name="name" required></li>
-    <li><input type="email" class="email"  placeholder="tshepomakama45@gmail.com" id="email" name="email" required></li>
-    <li><textarea class="msg"  id="msg" maxlength="500" placeholder="Write your message here." name="message" required></textarea></li>
-    <li><button type="submit" class="btn" id="btn">Get in touch</button></li>
-  </ul>
-</form>
-</section>
-	
+
+    <section id="contact">
+        <div class="contact_detail">
+            <h2 class="head">Contact me</h2>
+            <p class="detail">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                I’d love to help with it!
+            </p>
+        </div>
+        <form class="form" action="https://formspree.io/f/mayzgvgj" method="POST">
+            <input class="input" type="text" maxlength="30" name="text" required placeholder="Enter your name">
+            <input class="input" type="email" name="email" required placeholder="Enter your email">
+            <textarea class="input" maxlength="500" name="message" placeholder="Write your message here.." rows="7" required></textarea>
+            <button type="submit" class="submitt" >Get in touch</button>
+        </form>
```

Comment:
- [ ] You did well using the right button type here. :+1:  and also implement ting the hover state. :+1: But looking at your current implementation, when a user hovers over the button, the text is no more visible, I'll suggest you also change the text color on hover to that the `Get in touch` text can be seen. 

- - Your Button On hover:
![Screenshot from 2023-06-13 14-38-58](https://github.com/BilalLiaquat7/Portfolio/assets/107632780/c416c13b-3135-4c83-b50b-7ff5e9be05aa)